sdk/appconfiguration/Azure.Data.AppConfiguration/api/Azure.Data.AppConfiguration.netstandard2.0.cs

```diff
@@ -19,9 +19,11 @@ public partial class ConfigurationClient
         public override bool Equals(object obj) { throw null; }
         public virtual Azure.Response<Azure.Data.AppConfiguration.ConfigurationSetting> GetConfigurationSetting(Azure.Data.AppConfiguration.ConfigurationSetting setting, bool onlyIfChanged = false, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.Data.AppConfiguration.ConfigurationSetting> GetConfigurationSetting(Azure.Data.AppConfiguration.ConfigurationSetting setting, System.DateTimeOffset acceptDateTime, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Data.AppConfiguration.ConfigurationSetting> GetConfigurationSetting(string key, string label, string syncToken, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Yeah, we are still trying to figure out what exact set of scenarios the service team wants to support around this notification feature. I started with the minimally viable implementation and will expand if needed.

Considering that you get these three exact values with the change notification I expect this would be the most used overload (a bit of sample usage in https://gist.github.com/pakrym/5904304c62d5003fc75d5abbaf145b76#retreiving-an-updated-setting-after-an-eventgrid-notification).